src/fastmcp/server/sampling/run.py

```
@@ -0,0 +1,291 @@
+            )
+        else:
+            try:
+                result_value = await tool.run(tool_use.input)
+                tool_results.append(
+                    ToolResultContent(
+                        type="tool_result",
+                        toolUseId=tool_use.id,
+                        content=[TextContent(type="text", text=str(result_value))],
+                    )
+                )
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)